### PR TITLE
soletta-dev-app: Add iptables rules to allow access

### PR DESCRIPTION
**Pull request by brunobottazzini**
Signed-off-by: Bruno Bottazzini <bruno.bottazzini@intel.com>

### CODE REVIEW

**Comment by barbieri** [comment]
as @kad said in the ML, use both ip6tables and iptables (v4)

**Comment by brunobottazzini** [comment]
Updated with ip6tables and iptables

**Comment by ipuustin** [comment]
Looks good to me.

File: meta-ostro/recipes-soletta/dev-app/files/0001-Add-firewall-rules-to-allow-access.patch [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+diff --git a/scripts/units/soletta-dev-app-server.service.in b/scripts/units/soletta-dev-app-server.service.in
+index b18d1bb..412e618 100644
+--- a/scripts/units/soletta-dev-app-server.service.in
++++ b/scripts/units/soletta-dev-app-server.service.in
+@@ -3,7 +3,11 @@ Description=Run Soletta Development Application server
+ 
+ [Service]
+ StandardOutput=null
++ExecStartPre=/usr/sbin/iptables -w -A INPUT -p tcp --dport 80 -j ACCEPT
```
Comment (mythi) [inline meta-ostro/recipes-soletta/dev-app/files/0001-Add-firewall-rules-to-allow-access.patch]: this implies RDEPENDS to iptables.

Comment (brunobottazzini) [inline meta-ostro/recipes-soletta/dev-app/files/0001-Add-firewall-rules-to-allow-access.patch]: Indeed, I will add it

**Comment by okartau** [comment]
retest this please

**Comment by brunobottazzini** [comment]
@mythi, I have added the RDEPENDS as you suggested.

**Comment by ipuustin** [comment]
Wait, do not merge yet.

**Comment by ipuustin** [comment]
This should be integrated after soletta-dev-app is no longer enabled by default. See solettaproject/meta-soletta#52 and ostroproject/meta-ostro#83 for details.

**Comment by mythi** [comment]
@brunobottazzini can you rebase this please (on top of #83 which is merged)

**Comment by brunobottazzini** [comment]
@mythi, rebased

**Comment by mythi** [comment]
@brunobottazzini care to rebase and I'll merge then

**Comment by brunobottazzini** [comment]
@mythi rebased :)

**Comment by mythi** [comment]
test this please

**Comment by mythi** [comment]
test this please

**Comment by mythi** [comment]
test this please

**Pull request merged**